friends/src/App.js

```diff
@@ -0,0 +1,28 @@
+import React, { Component } from 'react';
+import axios from 'axios';
+import FriendsList from 'FriendsList';
+
+class App extends Component {
+  constructor() {
+    super();
+    this.state={
+      friendsData:[],
+      friend:""
+    };
+  }
+
```

Comment:
in order to get the friends data into your app, you will need to make an axios get request to ('http://localhost:5000/friends')  and .then do something with the response.